assets/src/dashboard/app/views/editorSettings/components.js

```diff
@@ -17,21 +17,23 @@
 /**
  * External dependencies
  */
-import styled from 'styled-components';
+import styled, { css } from 'styled-components';
 import { rgba } from 'polished';
 
 /**
  * Internal dependencies
  */
-import {
-  TypographyPresets,
-  StandardViewContentGutter,
-  Button,
-  TextInput,
-} from '../../../components';
+import { StandardViewContentGutter } from '../../../components';
```

Comment:
This is the only component that's left from the old set of components. It is a styled div that has some old styles (mentioned in another comment in `assets/src/dashboard/components/contentGutter/index.js`)

Reply:
Removed old styles here 🏁